lte/gateway/python/magma/mobilityd/rpc_servicer.py

```diff
@@ -58,21 +62,23 @@ def __init__(self, ip_address_manager: IPAddressManager,
 
         self._ip_address_man = ip_address_manager
 
-        # Load IPv4 block from the configurable mconfig file
+        # Load IPv4 and IPv6 blocks from the configurable mconfig file
         # No dynamic reloading support for now, assume restart after updates
+        logging.info('Adding ipv4 block')
         ipv4_block = _get_ip_block(ipv4_block_str)
         if ipv4_block is not None:
             try:
                 self.add_ip_block(ipv4_block)
             except OverlappedIPBlocksError:
-                logging.error("Overlapped IPv4 block: %s", ipv4_block)
+                logging.warning("Overlapped IPv4 block: %s", ipv4_block)
 
+        logging.info('Adding ipv6 block')
         ipv6_block = _get_ip_block(ipv6_block_str)
         if ipv6_block is not None:
             try:
                 self.add_ip_block(ipv6_block)
             except OverlappedIPBlocksError:
-                logging.error("Overlapped IPv6 block: %s", ipv6_block)
+                logging.warning("Overlapped IPv6 block: %s", ipv6_block)
```

Comment:
just as a cleanup can you pull the blocks logic out of the constructor into the client and update your constructor signature to take 2 optional ip network types